srfi-134.html

```diff
@@ -353,16 +353,16 @@ <h2 id="Conversion">Conversion</h2>
 
 <h1>Implementation</h1>
 
-<p>The sample implementation is in the <code>ideque-2list</code> subdirectory of the repository for
-this SRFI.  It is an implementation based on two Scheme lists, one
-for the front and another (in reverse element order) for the back.
-The cost of <code>ideque-front</code>, <code>ideque-back</code>,
-<code>ideque-remove-front</code>, and <code>ideque-remove-back</code>
-are amortized O(1).  An implementation based on finger trees may be
-added later.</p>
+<p>The sample implementation is in the <code>srfi</code> subdirectory of the repository for
+this SRFI.  It uses a two-stream banker's deque structure (see
+<em>Purely Functional Data Structures</em> by
+Chris Okasaki, p. 108) which provides amortized O(1) running time
+for basic deque operations.
+An implementation based on finger trees may be added later.</p>
 
 <h1>Copyright</h1>
 Copyright (C) John Cowan, Kevin Wortman (2015).  All Rights Reserved.
+Copyright (C) Wolfgang Corcoran-Mathe (2022).  All Rights Reserved.

```

Comment:
This copyright notice is for the SRFI document, not the code.  You've just changed one paragraph, so it's probably not right to add your name here.  What do you think?

Reply:
Oops, sorry about that. I'll revert that. I also found an arithmetic error in ideque-take, so standby for a new PR.